integration/kubernetes/confidential/lib.sh

```diff
@@ -31,6 +31,35 @@ kubernetes_wait_cc_pod_be_ready() {
 	kubectl wait --timeout=${wait_time}s --for=condition=ready pods/$pod_name
 }
 
+# Wait until the pod is 'Deleted'. Fail if it hits the timeout.
+#
+# Parameters:
+#	$1 - the sandbox ID
+#	$2 - wait time in seconds. Defaults to 120. (optional)
+#
+kubernetes_wait_cc_pod_be_deleted() {
+  local pod_name="${1}"
+  local wait_time="${2:-120}"
+
+  esudo kubectl wait --for=delete pod/$pod_name --timeout=${wait_time}s
```

Comment:
There's already some functions in `lib/common.bash`, such as `WaitForProcess` that could be used instead of this one.

If we want to add a function to specifically wait for a pod to be deleted, I'd suggest taking the bullet to add this everywhere needed, and also do the same for the case where we wait for a pod to be ready.